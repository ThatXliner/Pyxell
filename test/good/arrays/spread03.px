
print [...[1..3], ...[5, 6.0], -1/2..0]
print ['x', "qwerty", ...['a'..'e' step 2] step -2]
