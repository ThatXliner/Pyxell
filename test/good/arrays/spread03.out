[1, 2, 3, 5, 6, -0.5]
["x", "qwerty", "e", "a"]
